docs/framework/interop/registration-free-com-interop.md

```diff
@@ -54,11 +54,11 @@ Registration-free COM interop activates a component without using the Windows re
   
 ### Configuring COM components for registration-free activation  
   
-1.  For a COM component to participate in registration-free activation, it must be deployed as a side-by-side assembly. Side-by-side assemblies are unmanaged assemblies.  For more information, see "Using Side-by-side Assemblies" in the MSDN Library.  
+1.  For a COM component to participate in registration-free activation, it must be deployed as a side-by-side assembly. Side-by-side assemblies are unmanaged assemblies.  For more information, see "Using Side-by-side Assemblies" in Microsoft Docs.  
```

Comment:
let's change from
`see "Using Side-by-side Assemblies" in Microsoft Docs.`

to 
`see [Using Side-by-side Assemblies](https://msdn.microsoft.com/library/windows/desktop/aa376618.aspx).`